students/10th/soojungsohn/user/urls.py

```diff
@@ -2,10 +2,14 @@
 
 from .views import (
     SignUpView,
-    SignInView
+    SignInView,
+    FollowView,
+    UnFollowView
 )
 
 urlpatterns = [
-    path('/',SignUpView.as_view()),
-    path('/signin', SignInView.as_view()),
+    path('',SignUpView.as_view()),
```

Comment:
회원가입도 url path 주는게 좋을 것 같습니다.
```suggestion
    path(''/sign-up", SignUpView.as_view()),
```